proj4/src/main/scala/geotrellis/proj4/io/wkt/WktCS.scala

```diff
@@ -44,12 +44,23 @@ case class Projection(name: String, authority: Option[Authority])
 
 case class GeogCS(name: String, datum: Datum, primeM: PrimeM, angularUnit: UnitField, axes: Option[List[Axis]], authority: Option[Authority]) extends WktCS
 
-case class ProjCS(name: String, geogcs: GeogCS, projection: Projection, params: Option[List[Parameter]], linearUnit: UnitField, twinAxis: Option[TwinAxis], authority: Option[Authority]) extends WktCS
+case class ProjCS(name: String, geogcs: GeogCS, projection: Projection, params: Option[List[Parameter]], linearUnit: UnitField, twinAxis: Option[TwinAxis], extension: Option[Extension], authority: Option[Authority]) extends WktCS
 
 case class VertCS(name: String, vertDatum: VertDatum, unit: UnitField, axis: Option[Axis], authority: Option[Authority]) extends WktCS
 
 case class LocalCS(name: String, localDatum: LocalDatum, unit: UnitField, axisList: List[Axis], authority: Option[Authority]) extends WktCS
 
 case class GeocCS(name: String, datum: Datum, primeM: PrimeM, unit: UnitField, axis: Option[List[Axis]], authority: Option[Authority]) extends WktCS
 
-case class CompDCS(name: String, head: Any, tail: Any, authority: Option[Authority]) extends WktCS
\ No newline at end of file
+case class CompDCS(name: String, head: Any, tail: Any, authority: Option[Authority]) extends WktCS
+
+sealed trait Extension {
+  def name: String
+  def value: String
+}
+
+case class ExtensionProj4(value: String) extends Extension {
```

Comment:
I'm still wondering was it a good idea to make it a part of the parsing DSL or not.

Reply:
Can you describe why or why not? I think I'm lacking some project context to know more.

Reply:
https://github.com/locationtech/geotrellis/pull/3244#discussion_r427971659

Reply:
I'm also wondering, mb there is a better for Extention that is just anything and not Proj4?

Reply:
Including as part of the DSL seems fine, its what we do for all the other things we need to parse, so that feels consistent at least. 

IDK about better names. All the other parsers match the names being parsed out, like `Parser[GeocCS]` looks for `GEOCCS[]`. So what's here is consistent with the current naming convention. Kinda annoying that its such a generic word that could be confused in other contexts, but it seems pretty well isolated here at first glance, so :shrug: